### ISSUE TITLE
Project option is not accessible using the keyboard: A11y_.NET 9 template_.NET 9 template_Project_Keyboard.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/maui#26861]
### GitHub Tags
#A11yTCS; #A11ySev1; #Win32; #WCAG2.1.1; #SH_.NET9template_MAC_Dec2024; #DesktopApp; #.NET Core; #; #A11yMAS; #Keyboard; #MAC;#AILimited;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
MAC Version 15.2

### Repro Steps:

1. Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Tab till Hamburger menu and press enter, Projects options will screen will open.
4. Observe that Projects options are not accessible using the keyboard keys.

### Actual Result:
Project option is not accessible using the keyboard keys.

### Expected Result:
Project option should be accessible using the keyboard keys.

### User Impact:
keyboard users will not be able to access the controls and will not be able to access the other functionality.

### Attachment:
Project options

https://github.com/user-attachments/assets/24c225d4-33e7-4843-a7f9-446c3f7cf45a

**Comment by AnKushSingh05** [comment]
Issue is fixed.

https://github.com/user-attachments/assets/7e9f8c0c-81e3-40cc-a8a2-50f66e03ebd9
